qbft/instance.go

```diff
@@ -202,3 +212,8 @@ func (i *Instance) Encode() ([]byte, error) {
 func (i *Instance) Decode(data []byte) error {
 	return json.Unmarshal(data, &i)
 }
+
+// CanProcessMessages will return true if instance can process messages
+func (i *Instance) CanProcessMessages() bool {
+	return int(i.State.Round) < CutoffRound && !i.forceStop
```

Comment:
first check forceStop before comparing other vars. 
```suggestion
	return !i.forceStop && int(i.State.Round) < CutoffRound
```

Reply:
Also shouldn't we use forcestop to prevent the instance from broadcasting messages as well?

Reply:
we can give them a chance to broadcast imo

does @alonmuroch agree with me?

Reply:
why tho? if the (a term I just invented) "effectiveness threshold" has past, will broadcasting give them or the network any benefit ?
Anyway it seems it won't cause any harm as well apart from resources, so I'm ok with leaving this now until we know there's a good reason to stop them from broadcasting (resources or other).

Reply:
opened https://github.com/bloxapp/ssv-spec/issues/288